src/components/v2/values/chartValuesDiff/ChartValuesView.scss

```diff
@@ -97,7 +99,7 @@
         display: grid;
         grid-template-columns: 300px 1px calc(100% - 301px);
         transition: all 0.2s ease-out;
-        position: relative;
+        // position: relative;
```

Comment:
Have commented this so that the Deploy Chart cta which is `position: absolute` can align itself relatively from container instead of wrapper.

Reply:
Please remove this line